.gitignore

```diff
@@ -7,7 +7,6 @@
 .*.sw*
 .*.un~
 clover.xml
-composer.lock
```

Comment:
Why? All builds on Travis-CI use Composer update so what's the point of this file when is not involved in testing